StringExtensions.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Linq;
+
+namespace EntityManager
+{
+    public static class StringExtensions
+    {
+        public static string FirstCharToUpper(this string input)
+        {
+            if (!string.IsNullOrEmpty(input))
+            {
+                return input.First().ToString().ToUpper() + input.Substring(1);
+            }
+            return string.Empty;
+        }
+
+        public static string FirstCharToLower(this string input)
+        {
+            if (!string.IsNullOrEmpty(input))
+            {
+                return input.First().ToString().ToLower() + input.Substring(1);
+            }
+            return string.Empty;
+        }
+
+        public static string DropSuffix(this string input, string suffix)
+        {
+            if (input.EndsWith(suffix))
+            {
+                return input.Remove(input.Length - suffix.Length);
+            }
```

Comment:
you can remove brackets in 1 line if's<br>
            if (input.EndsWith(suffix))<br>
            return input.Remove(input.Length - suffix.Length);<br>

Reply:
Tim Corey advice was not to do so.